src/components/Darkmode/ToggleButton.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+
+import { useDispatch, useSelector } from "react-redux";
+import { changeDarkTheme } from "../../store/actions/exporterActions";
+
+// import Toggle from "react-toggle";
```

Comment:
I think you should remove unused import commanded.

Reply:
You are sharp! Thanks for reading my code.
I will remove this code as well and include it in the next pr.